cmd/run/run_test.go

```diff
@@ -732,6 +732,22 @@ func TestGRPCServingTLS(t *testing.T) {
 	})
 }
 
+func TestGRPCMaxMessageSize(t *testing.T) {
```

Comment:
@jon-whit I'm not following, how is this one testing the max message size change?

Reply:
That's residual. I didn't intend to commit that. My apologies. Will remove.